vendor/system_app.te

```diff
@@ -10,7 +10,9 @@ set_prop(system_app, dispcal_prop)
 # TimeKeep Java service
 allow system_app timekeep_vendor_data_file:dir create_dir_perms;
 allow system_app timekeep_vendor_data_file:file create_file_perms;
+allow system_app sysfs_rtc:lnk_file r_file_perms;
```

Comment:
The timekeep app will get its own domain so I will remove this later anyway, no point in adding this now

Reply:
Ok I will drop timekeep changes and wait for your changes

Reply:
Also, for your benefit, the `r_dir_file` macro already includes `r_file_perms` for links as well:
```
# te_macros.te:
define(`r_dir_file', `
allow $1 $2:dir r_dir_perms;
allow $1 $2:{ file lnk_file } r_file_perms;
')
```